Add basic config 1

Aichnerc opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)


**What is the new behavior (if this is a feature change)?**
- The basic configuration of a MDB React application has been set

Aichnerc committed:
Add basic React setup

This setup was achieved by molding the MDB boilerplate
to our own requirements and best practices.

Aichnerc committed:
Add comments

There were some missing comments that explain
what libs are imported.

Aichnerc committed:
Improve .gitignore

The .gitignore file got updated. Useless lines have been
removed and a general exclution of log files has been
established.

kleberbaum commented on .gitignore:
```diff
@@ -0,0 +1,20 @@
+# production
+/build
+
+# config
```
NOTE: Better comment would be all editor configs but vscode. xD

kleberbaum commented on src/App.js:
```diff
@@ -0,0 +1,36 @@
+// React router
+import { BrowserRouter as Router } from 'react-router-dom';
+
+// Components
```
NOTE: It would be nice to further describe the components.

kleberbaum commented on src/Routes.js:
```diff
@@ -0,0 +1,31 @@
+// React Router
+import { Route, Switch } from 'react-router-dom';
+
+// Components
```
NOTE: It would be nice to further describe the components.

kleberbaum commented on src/components/pages/HomePage.js:
```diff
@@ -0,0 +1,191 @@
+import MDBLogo from '../../assets/mdb-react-small.png';
+import AgencyLogo from '../../assets/agency-small.png';
+
+// Content
```
NOTE: It would be nice to further describe the contents.

kleberbaum reviewed (changes requested):
Add some comments, then it should be ok to merge.

Aichnerc committed:
Improve quality of comments

New commenting guidelines have been applied to improve
the quality of the comments and their readability.

Aichnerc committed:
Improve .gitignore

Improved the quality of comments.

Aichnerc committed:
Improve .gitignore

Improved the quality of comments.

kleberbaum commented on .gitignore:
```diff
@@ -1,19 +1,19 @@
 /build
 
-# config
+# All editor configs but vscode
```
NOTE: That's way better.

kleberbaum reviewed (approved):
It should be ok to merge.

Aichnerc committed:
Add multi-imports

This keeps the components very clean and
prevents redundant calling of elements of
the same component type (such as Atoms, ...).

kleberbaum reviewed (approved):
Still ok to merge.
